ArduPlane/control_modes.cpp

```diff
@@ -160,10 +160,25 @@ void Plane::reset_control_switch()
  */
 void Plane::autotune_start(void)
 {
-    gcs().send_text(MAV_SEVERITY_INFO, "Started autotune");
-    rollController.autotune_start();
-    pitchController.autotune_start();
-    yawController.autotune_start();
+    const bool tune_roll = g2.axis_bitmask.get() & int8_t(AutoTuneAxis::ROLL);
+    const bool tune_pitch = g2.axis_bitmask.get() & int8_t(AutoTuneAxis::PITCH);
+    const bool tune_yaw = g2.axis_bitmask.get() & int8_t(AutoTuneAxis::YAW);
+    if (tune_roll || tune_pitch || tune_yaw) {
+        gcs().send_text(MAV_SEVERITY_INFO, "Started autotune");
+        if (tune_roll) { 
+            rollController.autotune_start();
+        }
+        if (tune_pitch) { 
+            pitchController.autotune_start();
+        }
+        if (tune_yaw) { 
+            yawController.autotune_start();
+        }
+        gcs().send_text(MAV_SEVERITY_INFO, "Autotuning %s%s%s", tune_roll?"roll ":"", tune_pitch?"pitch ":"", tune_yaw?"yaw":"");
+    } else {
+        gcs().send_text(MAV_SEVERITY_INFO, "No axis selected for tuning!");
+        return;
+    }        
```

Comment:
Don't need the return on 180.
```suggestion
        gcs().send_text(MAV_SEVERITY_INFO, "Autotuning %s%s%s", tune_roll?"roll ":"", tune_pitch?"pitch ":"", tune_yaw?"yaw":"");
        return;
    }
    gcs().send_text(MAV_SEVERITY_INFO, "No axis selected for tuning!");
```
or just
```suggestion
        gcs().send_text(MAV_SEVERITY_INFO, "Autotuning %s%s%s", tune_roll?"roll ":"", tune_pitch?"pitch ":"", tune_yaw?"yaw":"");
    } else {
        gcs().send_text(MAV_SEVERITY_INFO, "No axis selected for tuning!");
    }        
```